### PR TITLE
feat/29 sign in with token authentication

### PR DESCRIPTION
**Pull request by Goketech**
​

## Description

Created token authentication

## Related Issue (Link to issue ticket)

#29 

​

## Motivation and Context

​This adds a new feature to the project

## How Has This Been Tested?

​Tests were written for this endpoint

## Screenshots (if appropriate - Postman, etc):
Screenshot (430)

​
Screenshot (431)

Screenshot (429)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
      ​

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by joboy-dev — DISMISSED** [review]
@bolexs this is fine

**Review by SundayMba — DISMISSED** [review]

**Comment by SundayMba** [comment]
@Dev-wonderful okay.

**Comment by Goketech** [comment]
@zxenonx

**Comment by zxenonx** [comment]
> @zxenonx

We need a screenshot of the endpoint testing on Postman

File: api/utils/email_service.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import smtplib
+from api.utils.settings import settings
+from fastapi import HTTPException
+
+def send_mail(to: str, subject: str, body: str):
+    '''Function to send email to a user either as a regular test or as html file'''
+    print(f"Sending email to {to}")
+    print(settings.MAIL_SERVER, settings.MAIL_PORT, settings.MAIL_USERNAME, settings.MAIL_PASSWORD, settings.MAIL_FROM)
```
Comment (zxenonx) [inline api/utils/email_service.py:8]: remove the print statements

**Review by zxenonx — CHANGES_REQUESTED** [review]
Thanks @Goketech 
Add a screenshot on which you test the endpoint on Postman

**Comment by Goketech** [comment]
@zxenonx changes applied

**Review by joboy-dev — APPROVED** [review]

**Pull request closed without merging**